modules/ROOT/pages/external-organization-access.adoc

```diff
@@ -0,0 +1,39 @@
+= External Organization Access
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The External Access feature allows separate organizations to collaborate in the Anypoint Platform. For example, organizations can share API specifications, connectors, and other assets in the Anypoint Exchange with each other, without making them public.
+
+Administrators control which organizations their users can collaborate with by maintaining a list of trusted organizations in Access Management. Adding an entry to this list means that you trust the other organization and its users, and you acknowledge that you are enabling your users to grant permission to and be granted permissions by the other organization.
+
+Trust must be established both ways before permissions can be granted. For the users of Organization A to share with the users of Organization B, Organization A must list Organization B as a trusted organization, and Organization B must also list Organization A. If either Organization A or Organization B removes the other, all shared permissions are revoked.
+
+Permissions shared between organizations are granted at the organization level. For example, if Organization A grants Organization B permission to view an asset, it grants that permission to every user in Organization B. Organization A does not receive any information about Organization B's users, sub-organizations, or internal layout.
+
```

Comment:
Just FYI, Will refers to "granting organization" and "receiving organization," but that is in the focused context of one-way collaboration rather than both ways. Anyway, I don't know if you guys have compared notes, but you might want to, re. terminology use.

Reply:
We chatted about it. Granting and receiving is only on the Exchange end of things, so he is okay to use that terminology. We also discussed his changes for two-way, so those should be implemented soon if he hasn't added them already.